apps/web/src/pages/Applications/ApplicationProfilePage/ApplicationProfilePage.tsx

```diff
@@ -75,16 +100,195 @@ export const getPageInfo: GetApplicationPageInfo = ({
+        application?.poolAdvertisement,
+      );
+      if (missingLanguageRequirements.length > 0) {
+        const requirements = missingLanguageRequirements.map((requirement) => (
+          <li key={requirement.id}>{intl.formatMessage(requirement)}</li>
+        ));
+        toast.error(
+          intl.formatMessage(
+            {
+              defaultMessage:
+                "You are missing the following language requirements: {requirements}",
+              id: "CPHTk9",
+              description:
+                "Error message when a user does not meet a pools language requirements",
+            },
+            { requirements },
+          ),
+        );
+      }
+    }
+
+    return false;
+  };
+
+  const handleUpdate: SectionProps["onUpdate"] = (userId, userData) => {
+    return executeUpdateMutation({
+      id: userId,
+      user: userData,
+    }).then((res) => res.data?.updateUserAsUser);
+  };
+
+  const sectionProps = {
+    user,
+    isUpdating,
+    onUpdate: handleUpdate,
+  };
 
-  return <Heading data-h2-margin-top="base(0)">{pageInfo.title}</Heading>;
+  return (
+    <>
+      <Heading data-h2-margin-top="base(0)">{pageInfo.title}</Heading>
+      <p>
+        {intl.formatMessage({
+          defaultMessage:
+            "This step includes all of your profile information required for this application. Each section has a series of required fields that must be completed to move on to the next step. Completing this profile information will only happen once, and the information you provide here will auto-populate on applications you submit moving forward.",
+          id: "D9Elm9",
+          description: "Application step to complete your profile, description",
+        })}
+      </p>
+      <div
+        data-h2-margin="base(x2, 0)"
+        data-h2-display="base(flex)"
+        data-h2-flex-direction="base(column)"
+        data-h2-gap="base(x1, 0)"
+      >
+        <ErrorSummary user={user} application={application} />
+        <PersonalInformation {...sectionProps} />
+        <WorkPreferences {...sectionProps} />
+        <DiversityEquityInclusion {...sectionProps} />
+        <GovernmentInformation {...sectionProps} />
+        <LanguageProfile {...sectionProps} application={application} />
+      </div>
+      <Separator
+        orientation="horizontal"
+        data-h2-background-color="base(gray.lighter)"
+        data-h2-margin="base(x2, 0)"
+        decorative
+      />
+      <div
+        data-h2-display="base(flex)"
+        data-h2-gap="base(x.25, x.5)"
+        data-h2-flex-wrap="base(wrap)"
+        data-h2-flex-direction="base(column) l-tablet(row)"
+        data-h2-align-items="base(flex-start) l-tablet(center)"
+      >
+        <form onSubmit={handleNavigation}>
+          <Button
+            type="submit"
+            color="primary"
+            mode="solid"
+            disabled={submitting}
+          >
+            {intl.formatMessage({
+              defaultMessage: "Let's go!",
```

Comment:
The description of this string, **Link text to begin the application process**, makes it clear what the button is doing, but the actual copy, **Let's go!**, does not make it clear what action a user is taking.

Reply:
Updated in [update submit button text](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6296/commits/0cc516e66b416d8bbf560e04232d59d95a03ce85)